pkgs/applications/networking/instant-messengers/webcord/webcord-vencord/default.nix

```diff
@@ -0,0 +1,26 @@
+# Webcord with Vencord.
+# Utilizes the Vencord Chrome extension.
+{
+  webcord,
+  substituteAll,
+  callPackage,
+  lib,
+}: let
+  vencord = callPackage ./vencord.nix {};
+in
+  webcord.overrideAttrs (old: {
+    patches =
+      (old.patches or [])
+      ++ [
+        (substituteAll {
+          src = ./add-extension.patch;
+          inherit vencord;
+        })
+      ];
+    meta = with lib;
+      old.meta
+      // {
+        license = licenses.gpl3Only;
+        maintainers = with maintainers; [FlafyDev NotAShelf];
+      };
+  })
```

Comment:
```suggestion
}:
let
  vencord = callPackage ./vencord.nix {};
in
  webcord.overrideAttrs ({ patches ? [ ], ... }: {
    patches = patches ++ [
      (substituteAll {
         src = ./add-extension.patch;
         inherit vencord;
      })
    ];

    meta = old.meta // (with lib; {
      license = licenses.gpl3Only;
      maintainers = with maintainers; [FlafyDev NotAShelf];
    });
  })
```

Reply:
is that nixpkgs-fmt? if so then i made sure to format with it